defcon/plugins/endpoint.py

```diff
@@ -0,0 +1,102 @@
+"""DefCon Prometheus Alertmanager plugin."""
+import requests
+import logging
+
+from defcon.plugins import base
+
+
+class EndpointPlugin(base.Plugin):
+    """DefCon Endpoint plugin.
+
+    set an endpoint to get a defcon level.
+    The endpoint should have the info {...'defcon':[1-5],...}
+    Another endPoint can be setup to force the level
+
+    Config:
+    ```python
+    {
+        'title': 'Defcon Production',
+        'url': 'http://defcon.com/api/simple/production/', // Url to endpoint
+        'link_status': 'http://defcon.com/status/production/ // Url to endpoint info
+        'force_url': 'http://defcon.com/api/simple/force/, // Url to force endpoint
+    }
+    ```
+    """
+
+    def __init__(self, config=None):
+        """Create an instance of the plugin."""
+        super(EndpointPlugin, self).__init__(config)
+        if config is None:
+            config = {}
+
+        self.url = config.get('url')
+        self.force_url = config.get('force_url', 'http://defcon.com/status')
```

Comment:
this looks like a weird default, None would probably be better.

Reply:
weird default => None, done